dashboard/v1.1/src/pages/Ping/index.tsx

```diff
@@ -1,7 +1,151 @@
+  const [value] = useState(routesDetails?.testServicesRoutes[0]);
+  const [inputValue, setInputValue] = useState('');
+  const [showCharts, setShowCharts] = useState(false);
+  const [pingData, setPingData] = useState<showChartsDataParam>();
+
+  const { response, error } = useFetch<service_states>(
+    `${HOST_IP}/service-state`
+  );
+
+  useEffect(() => {
+    fetch(`${HOST_IP}/routes-summary`)
+      .then(res => res.json())
+      .then((response: { status: string; data: RoutesSummary }) => {
+        console.log(response.data);
+        setRoutesDetails(response.data);
+      });
+  }, []);
+
+  async function getChartsData(v: any) {
+    var res = '';
+    if (v.startsWith('https://www.')) {
+      res = v.replace('https://www.', '');
+    } else if (v.startsWith('https://')) {
+      res = v.replace('https://', '');
+    }
+
+    try {
+      const response = await fetch(
+        `${HOST_IP}/query?timeSeriesPath=storage/ping/chunk_ping_${res}`
+      );
+      const matrix = (await response.json()) as APIQueryResponse;
+      var formatdata = format(matrix.data);
+      setPingData(formatdata);
+      setShowCharts(true);
+    } catch (e) {
+      console.error(e);
+    }
+  }
+
+  if (error) {
+    return <Alert severity="error">Unable to reach the service: error</Alert>;
+  }
+  if (response.data) {
+    // const states: service_states = response.data;
+
+    const options =
+      routesDetails?.testServicesRoutes !== undefined
+        ? routesDetails.testServicesRoutes
+        : ['Please fill routes'];
+
+    return (
+      <Card>
+        <CardContent>
+          <div>
+            <h4>Ping</h4>
+            <div style={{ float: 'right', marginTop: '-45px' }}>
+              <Autocomplete
+                value={value}
+                onChange={(event, newValue) => {
+                  console.log('newValue: ', newValue);
+                }}
+                inputValue={inputValue}
+                onInputChange={(event, newInputValue) => {
+                  console.log('newInputValue: ', newInputValue);
+                  setInputValue(newInputValue);
+                  getChartsData(newInputValue);
+                }}
+                id="controllable-states-demo"
+                options={options}
+                style={{ width: 300 }}
+                renderInput={params => (
+                  <TextField
+                    {...params}
+                    label="Select Route"
+                    variant="outlined"
+                  />
+                )}
+              />
+            </div>
+          </div>
+          <br />
+          <hr />
+          <div>
+            {pingData !== undefined && showCharts === true ? (
+              <Ping
+                min={pingData.pingMin}
+                mean={pingData.pingMean}
+                max={pingData.pingMax}
+              />
+            ) : null}
+          </div>
+        </CardContent>
+      </Card>
+    );
+  }
+
+  return <Spinner color="info" />;
```

Comment:
I think, this will be better: `<Alert severity="info">Fetching from sources <Spinner color="info" /> </Alert>;`